templates/components/products/product-view.html

```diff
@@ -230,6 +230,7 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
             {{/if}}
         </div>
         {{> components/common/share}}
+        {{#if settings.facebook_like_button}}{{{settings.facebook_like_button}}}{{/if}}
```

Comment:
Should this go in the `common/share` component? Do we only want it to show on this page, or anywhere we have the share buttons?

Reply:
Facebook like button isn't the same as the other share buttons it's an iframe and not an icon for one.  I think it's good to keep them separate.